services/api/pepc_boost_test.go

```diff
@@ -226,813 +318,718 @@ func TestCheckTxAndSenderValidity(t *testing.T) {
-		parentHash: parentHash,
-		payloadAttributes: beaconclient.PayloadAttributes{
-			PrevRandao: prevRandao,
+		{
+			description: "first set of txs is higher",
+			firstTobTxs: []*gethtypes.Transaction{
+				gethtypes.NewTx(&gethtypes.LegacyTx{
+					Nonce:    1,
+					GasPrice: big.NewInt(1),
+					Gas:      1,
+					To:       &uniswapV2Addr,
+					Value:    big.NewInt(1),
+					Data:     []byte("tx1"),
+				}),
+				gethtypes.NewTx(&gethtypes.LegacyTx{
+					Nonce:    2,
+					GasPrice: big.NewInt(2),
+					Gas:      2,
+					To:       &backend.relay.relayerPayoutAddress,
+					Value:    big.NewInt(2),
+					Data:     []byte(""),
+				}),
+			},
+			secondTobTxs: []*gethtypes.Transaction{
+				gethtypes.NewTx(&gethtypes.LegacyTx{
+					Nonce:    3,
+					GasPrice: big.NewInt(3),
+					Gas:      3,
+					To:       &uniswapV2Addr,
+					Value:    big.NewInt(3),
+					Data:     []byte("tx3"),
+				}),
+				gethtypes.NewTx(&gethtypes.LegacyTx{
+					Nonce:    4,
+					GasPrice: big.NewInt(5),
+					Gas:      12,
+					To:       &backend.relay.relayerPayoutAddress,
+					Value:    big.NewInt(1),
+					Data:     []byte(""),
+				}),
+			},
+			nextSentIsHigher: false,
 		},
-		withdrawalsRoot: phase0.Root(withdrawalsRoot),
 	}
 
-	// Test 3: Payout tx is to the wrong address
-	req := new(common.TobTxsSubmitRequest)
-	addr1 := common2.HexToAddress("0xB9D7a3554F221B34f49d7d3C61375E603aFb699e")
-
-	tx8 := gethtypes.NewTx(&gethtypes.LegacyTx{
-		Nonce:    3,
-		GasPrice: big.NewInt(3),
-		Gas:      3,
-		To:       &addr1,
-		Value:    big.NewInt(3),
-		Data:     []byte("tx6"),
-	})
-	tx9 := gethtypes.NewTx(&gethtypes.LegacyTx{
-		Nonce:    4,
-		GasPrice: big.NewInt(5),
-		Gas:      12,
-		To:       &addr1,
-		Value:    big.NewInt(5),
-		Data:     []byte(""),
-	})
-	tx8byte, err := tx8.MarshalBinary()
-	require.NoError(t, err)
-	tx9byte, err := tx9.MarshalBinary()
-	require.NoError(t, err)
-	txs := bellatrixUtil.ExecutionPayloadTransactions{Transactions: []bellatrix.Transaction{tx8byte, tx9byte}}
-	req = &common.TobTxsSubmitRequest{
-		ParentHash: parentHash,
-		TobTxs:     txs,
-		Slot:       headSlot + 1,
-	}
-	fmt.Printf("Marshalling request to json!!")
-	jsonReq, err := req.MarshalJSON()
-	require.NoError(t, err)
-	fmt.Printf("Unmarshalling request from json!!")
-
-	rr := backend.requestBytes(http.MethodPost, path, jsonReq, map[string]string{
-		"Content-Type": "application/json",
-	})
-	require.Equal(t, http.StatusBadRequest, rr.Code)
-	require.Contains(t, rr.Body.String(), "We require a payment tx to the relayer along with the TOB txs!")
+	for _, c := range cases {
+		t.Run(c.description, func(t *testing.T) {
+			backend := newTestBackend(t, 1)
+
+			headSlot := uint64(32)
+			submissionSlot := headSlot + 1
+			submissionTimestamp := 1606824419
+
+			// Payload attributes
+			payloadJSONFilename := "../../testdata/submitBlockPayloadCapella_Goerli.json.gz"
+
+			parentHash := "0xbd3291854dc822b7ec585925cda0e18f06af28fa2886e15f52d52dd4b6f94ed6"
```

Comment:
third time we have used these constants. can we def them at the top of the file ?